### DISCUSSION TITLE
Support subtitles

### DISCUSSION THREAD

**Discussion by homocomputeris** [PaulWoitaschek/Voice#1671] [Ideas]
Many typically audio containers such as ogg, mka, mp4 (m4b, m4a, etc.) can include subtitles.

It would be great if Voice could display them because such a feature is very useful for language learning.

**Comment by Geremia** [comment]
[VLC](https://www.videolan.org/) (at least the desktop version; I'm not sure about Android) supports this:

> <img width="309" alt="Image" src="https://github.com/user-attachments/assets/b30af361-327a-45ae-b508-5a54f34ea588" />
> 
> I changed the suffix to *.mp4 to upload here, but usually such files are *.m4a (MPEG **a**udio) or *.m4b (audio**b**ook):
> 
> https://github.com/user-attachments/assets/f0a244b7-875a-43ed-adf6-1f97f9996f31

<sup>[source](https://github.com/koreader/koreader/issues/14086#issuecomment-3138104785)</sup>

**Comment by secureman** [comment]
This a great idea the app is already awesome, if you can add word highlight, like in the elevenreader app that would make this app the best.

**Comment by FinnX0** [comment]
I sometimes download my audiobooks from YouTube, I think it's also possible to download the subtitle file. I think it would be really useful to be able to add the function of reading those subtitles in sync with the audiobook. I think the chapter division might be a bit of an issue, but maybe it could be amended even if I'd have to split them manually in the app, or just add up the chapter time to the total time of the audiobook, so it can stay in sync.
